content/docs/csidriver/_index.md

```diff
@@ -22,7 +22,7 @@ The CSI Drivers by Dell EMC implement an interface between [CSI](https://kuberne
 | CentOS        |     7.8, 7.9     |      7.8, 7.9       |     7.8, 7.9     |      7.8, 7.9     |     7.8, 7.9     |
 | SLES          |        15SP2        |        15SP2        |       15SP2      |         15SP2     |       15SP2      |
```

Comment:
Need to change to SP3

Reply:
In this table we should also update the access modes to add RWOP for the drivers that implemented it.
@shanmydell please double check the pmax microcode/unisphere versions specified in the Support storage platforms section.

Reply:
addressed